### ISSUE TITLE
Create folder/file structure similar to Ambiance

### ISSUE THREAD

**Issue by robertpainsi** [rbrito/pkg-mediterranean-gtk-themes#29]
To be consistent with the main themes we should also provide a similar folder/file structure. Mainly I'm talking about an app folder with all the app specific files.

```
/usr/share/themes/Ambiance
```

This should be done _after_ the current refactoring but I would suggest that it's still on the milestone.

**Comment by robertpainsi** [comment]
Or even better and cleaner for the sass version https://github.com/shimmerproject/Numix/tree/sass/gtk-3.0/scss

**Comment by rbrito** [comment]
Hi.

On Feb 10 2015, Robert Painsi wrote:

> Or even better and cleaner for the sass version
> https://github.com/shimmerproject/Numix/tree/sass/gtk-3.0/scss

You guys have done a great job discussing the maintainability of the theme,
but I have a question: there were some pull requests that became closed and
I have lost track of what I should merge or not merge.

Can you guys give me "most current" what should I merge now? I am trying to
catch up with some projects of mine and I would appreciate some hints.

Thanks for all the work,

## 

Rogério Brito : rbrito@{ime.usp.br,gmail.com} : GPG key 4096R/BCFCAAAA
http://cynic.cc/blog/ : github.com/rbrito : profiles.google.com/rbrito
DebianQA: http://qa.debian.org/developer.php?login=rbrito%40ime.usp.br

**Comment by robertpainsi** [comment]
Yeah, since the porting progress was very fast, pull requests were opened and closed before they even could get reviewed :smile_cat:

I haven't created a new PR yet because there are two options we can choose from
- Merge https://github.com/robertpainsi/pkg-mediterranean-gtk-themes/tree/sass directly into master. I don't know how many of you have reviewed the code yet. https://github.com/rbrito/pkg-mediterranean-gtk-themes/pull/31 or https://github.com/robertpainsi/pkg-mediterranean-gtk-themes/commit/9b3bd2c15c38e4b91f891688b8248fca8cdcbca3 was the last commit which allowed a automated way to check if nothing has been changed. But since there aren't too many people using this repository, issues and other things will be found more quickly and can be resolved before this repository gains a bigger audience. A compiled version is maybe missing... or not. Normally you don't provide compiled versions in a github repo. On the other hand, it's not an app, it's a theme... However...
- Create a new branch sass and do all further work there. But this branch can be easily overlooked. I also had luck to find the Numix sass version because I was exactly looking for sass.

I'm happy either way. Nevertheless someday we have to merge the sass branch into the master :smirk_cat: (if you all agree of course!)

**Comment by robertpainsi** [comment]
But in the meantime you could create a sass branch in your repo. So I can create PRs for that branch and you guys can review them and also keep up with the changes that have been made. I shouldn't be the only one who knows what has been done during the refactoring process. :smile_cat:

I also appreciate some feedback. So you guys can share your thoughts, maybe see some improvements, or mistakes I made. We can also discuss further steps which we should take. But without PRs this isn't possible, so "gimie sass branch plz" :smirk_cat:
